src/css/clustermanagement.css

```diff
@@ -253,3 +253,73 @@ div.nodetooltip {
 .advanced-options-toggle {
   margin-right: 6px;
 }
+
+/**----------------------------------------------*/
```

Comment:
I'll put these here because I don't want to search for the appropriate place for them:
* Nodes in the legend are smaller than in the mockup
* Headings of the sections are too big
* Font color is different
* We don't have leader and follower icons anymore
* Color of the lines representing the links should be the secondary color and not the tertiary (the green one) which you've used
* The thikness of the lines should be 4px probaly
* The recovering link type is missing
* The sync state rows are quite bumped into each other
* The legend panel is positioned too low and covers the button that opens it a bit.

Reply:
And one more thing here. Cluster node hexagons now should have rounded corners. Look mockups and the notes there carefully.